.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Do we know what's installed on this machine? Or is it better to run the build in a docker container if that's possible? So that we have more control over the build environment.

Reply:
It is the base image of ubuntu 18.04 LTS. I believe it is possible to run in a docker container. As this is a standalone Java project and has no system dependencies (is there any system dependency?) it should be fine running without a container.

Reply:
I am on the same page to use docker image for building (which uses open jdk version). I have updated the PR.